src/db/spanner/models.rs

```diff
@@ -1204,7 +1204,7 @@ impl SpannerDb {
             Sorting::Index => format!("{} ORDER BY sortindex DESC", query),
             Sorting::Newest => format!("{} ORDER BY modified DESC", query),
             Sorting::Oldest => format!("{} ORDER BY modified ASC", query),
-            _ => query,
```

Comment:
Kinda wondering if this might be better as:
```
query = match sort {
        query = match sort {
            Sorting::Index | Sorting::Newest => format!("{} ORDER BY sortindex DESC", query),
            _ => format!("{} ORDER BY sortindex ASC", query),
};
```

Reply:
Hm, Index needs to be ORDER BY sortindex and Newest needs to be ORDER BY modified though, so this won't work.